tests/Fixer/ClassNotation/FinalClassFixerTest.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Tests\Fixer\ClassNotation;
+
+use PhpCsFixer\Tests\Test\AbstractFixerTestCase;
+
+/**
+ * @author Filippo Tessarotto <zoeslam@gmail.com>
+ *
+ * @internal
+ *
+ * @covers \PhpCsFixer\Fixer\ClassNotation\FinalClassFixer
+ */
+final class FinalClassFixerTest extends AbstractFixerTestCase
+{
+    /**
+     * @param string      $expected
+     * @param null|string $input
+     *
+     * @dataProvider provideFixCases
+     */
+    public function testFix($expected, $input = null)
+    {
+        $this->doTest($expected, $input);
+    }
+
+    public function provideFixCases()
+    {
+        return [
+            ['<?php /** @Entity */ class MyEntity {}'],
+            ['<?php use Doctrine\ORM\Mapping as ORM; /** @ORM\Entity */ class MyEntity {}'],
+            ['<?php abstract class MyAbstract {}'],
+            ['<?php trait MyTrait {}'],
+            [
+                '<?php final class MyClass {}',
+                '<?php class MyClass {}',
+            ],
+            [
+                '<?php final class MyClass extends MyAbstract {}',
+                '<?php class MyClass extends MyAbstract {}',
+            ],
+            [
+                '<?php final class MyClass implements MyInterface {}',
+                '<?php class MyClass implements MyInterface {}',
+            ],
+            [
+                '<?php /** @codeCoverageIgnore */ final class MyEntity {}',
+                '<?php /** @codeCoverageIgnore */ class MyEntity {}',
+            ],
```

Comment:
Can we add some tests for:

- Multiple classes in one file
- Interfaces (don't get changed obviously :D)
- PHPDoc with mentions of "Entity" but not an actual annotation

Reply:
Thanks @Slamdunk, anyone know of a way of marking comments as resolved without deleting them? 😆

Reply:
Approve the PR?

Reply:
Nope, but I get your approval :stuck_out_tongue:

Reply:
@ntzm - yes, with hand made comment like `changes applied` ;)